formatTest/typeCheckedTests/input/basics.re

```diff
@@ -22,6 +22,18 @@ let l1 =
 let l2 =
   [Some(1), None, Some(2)] |> List.map(optParam(~v =?_, ())) |> List.length;
 
+let argIsUnderscore1 = _ => 34;
+
+let argIsUnderscore2 = (_ => 34);
+  
+let argIsUnderscore3 = _ : int => 34;
```

Comment:
Side note: I realize return value type annotations look a little strange without parens on the argument. Whether or not it's an underscore.

Reply:
@jordwalke I also realize this problem. What about just refmt it as `(_) : int => 34` https://github.com/facebook/reason/issues/1977
It's a little bit weird when the arg is `_` tho.